.github/workflows/pr-operator.yml

```diff
@@ -258,6 +255,23 @@ jobs:
           platforms: ${{ matrix.platform }}
           push: false
           tags: "${{ env.OPERATOR_IMAGE_REPOSITORY }}:latest-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }},${{ env.OPERATOR_IMAGE_REPOSITORY }}:${{ env.OPERATOR_VERSION }}-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }}"
+          load: true
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@0.14.0
+        with:
+          scan-type: image
+          image-ref: ${{ env.OPERATOR_IMAGE_REPOSITORY }}:latest-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }}
+          format: "cosign-vuln"
+          output: "operator-cosignvuln.json"
+
+      - name: Run Trivy SBOM generator
+        uses: aquasecurity/trivy-action@0.14.0
+        with:
+          scan-type: image
+          image-ref: ${{ env.OPERATOR_IMAGE_REPOSITORY }}:latest-${{ steps.setup-build-step.outputs.platform_os }}-${{ steps.setup-build-step.outputs.platform_arch }}
+          format: "cyclonedx"
```

Comment:
Can we swap to spdx as this is the SBOM format RH is aligning to?

https://www.redhat.com/en/blog/future-red-hat-security-data